refinery/config/settings/base.py

```diff
@@ -615,3 +615,10 @@ def get_setting(name, settings=local_settings, default=None):
 
 # temporary feature toggle for using S3 as user data file storage backend
 REFINERY_S3_USER_DATA = get_setting('REFINERY_S3_USER_DATA', default=False)
+
+# ALLOWED_HOSTS required in 1.8.16 to prevent a DNS rebinding attack.
+ALLOWED_HOSTS = get_setting("ALLOWED_HOSTS")  # NOQA: F405
```

Comment:
required both for dev and prod, so moved to base

Reply:
I don't think `# NOQA: F405` is necessary here (http://flake8.pycqa.org/en/3.1.1/user/error-codes.html)